<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddRuleWindow</class>
 <widget class="QDialog" name="AddRuleWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>738</width>
    <height>237</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add New Rule</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="1" column="1">
    <widget class="QPushButton" name="buttonAddRule">
     <property name="text">
      <string>Add Rule</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="buttonCancel">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="3">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" colspan="4">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>4</number>
     </property>
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>If</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QGroupBox {background: white; border-radius: 8px; border: 1px solid gray;}</string>
       </property>
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QRadioButton" name="checkLastPrice">
          <property name="text">
           <string>Last Price</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="checkMarketBuy">
          <property name="text">
           <string>Market Buy</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="checkMarketSell">
          <property name="text">
           <string>Market Sell</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="checkMarketHigh">
          <property name="text">
           <string>Market High</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="checkMarketLow">
          <property name="text">
           <string>Market Low</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Goes</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QGroupBox {background: white; border-radius: 8px; border: 1px solid gray;}</string>
       </property>
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QRadioButton" name="checkGoesAbove">
          <property name="text">
           <string>More</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="checkGoesBelow">
          <property name="text">
           <string>Less</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Than</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>4</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <widget class="QLabel" name="label_52">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: darkgreen</string>
         </property>
         <property name="text">
          <string>$</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="thanValue">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>0</height>
          </size>
         </property>
         <property name="prefix">
          <string/>
         </property>
         <property name="decimals">
          <number>5</number>
         </property>
         <property name="minimum">
          <double>0.010000000000000</double>
         </property>
         <property name="maximum">
          <double>999999999.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QGroupBox {background: white; border-radius: 8px; border: 1px solid gray;}</string>
       </property>
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QRadioButton" name="checkSellAmount">
          <property name="text">
           <string>Sell Amount</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="checkBuyAmount">
          <property name="text">
           <string>Buy Amount</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="checkSellAllIn">
          <property name="text">
           <string>Sell All In</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="checkBuyAllIn">
          <property name="text">
           <string>Buy All In</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="checkSellHalfIn">
          <property name="text">
           <string>Sell Half In</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="checkBuyHalfIn">
          <property name="text">
           <string>Buy Half In</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_19">
       <property name="spacing">
        <number>4</number>
       </property>
       <item>
        <widget class="QLabel" name="btcLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: #996515</string>
         </property>
         <property name="text">
          <string>à¸¿</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="btcValue">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>0</height>
          </size>
         </property>
         <property name="prefix">
          <string/>
         </property>
         <property name="decimals">
          <number>8</number>
         </property>
         <property name="minimum">
          <double>0.010000000000000</double>
         </property>
         <property name="maximum">
          <double>999999999.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>buttonCancel</sender>
   <signal>clicked()</signal>
   <receiver>AddRuleWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>422</x>
     <y>217</y>
    </hint>
    <hint type="destinationlabel">
     <x>433</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonAddRule</sender>
   <signal>clicked()</signal>
   <receiver>AddRuleWindow</receiver>
   <slot>buttonAddRule()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>304</x>
     <y>216</y>
    </hint>
    <hint type="destinationlabel">
     <x>304</x>
     <y>261</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>buttonAddRule()</slot>
 </slots>
</ui>
